exporter/splunkhecexporter/client_test.go

```diff
@@ -1266,15 +1266,15 @@ func Test_PushMetricsData_Summary_NaN_Sum(t *testing.T) {
 func TestReceiveMetricsWithCompression(t *testing.T) {
 	cfg := NewFactory().CreateDefaultConfig().(*Config)
 	cfg.MaxContentLengthMetrics = 1800
-	request, err := runMetricsExport(cfg, createMetricsData(1, 100), 1, false, t)
+	request, err := runMetricsExport(cfg, createMetricsData(1, 100), 2, false, t)
```

Comment:
We need to expect two batches because when the metrics are encoded as Splunk HTTP events they're larger than the set `MaxContentLengthMetrics` value.